From 1f4a850a91508a017d972d0af9f998d72138b5e5 Mon Sep 17 00:00:00 2001
From: Artur Szafraniec <ganggg111@gmail.com>
Date: Tue, 29 Sep 2026 15:50:20 +0200
Subject: [PATCH 04/29] mobileshell: use Milou.ResultsView from Plasma 6.7

Milou 6.7 renamed ResultsListView to ResultsView, so the Folio home screen
failed to load with "Milou.ResultsListView is not a type".  Same change
as upstream plasma-mobile 6.7.

Signed-off-by: Artur Szafraniec <ganggg111@gmail.com>
---
 components/mobileshell/qml/widgets/krunner/KRunnerScreen.qml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/components/mobileshell/qml/widgets/krunner/KRunnerScreen.qml b/components/mobileshell/qml/widgets/krunner/KRunnerScreen.qml
index 58081323..8cbc442b 100644
--- a/components/mobileshell/qml/widgets/krunner/KRunnerScreen.qml
+++ b/components/mobileshell/qml/widgets/krunner/KRunnerScreen.qml
@@ -98,7 +98,7 @@ MouseArea {
             Layout.rightMargin: Kirigami.Units.gridUnit
             Layout.alignment: Qt.AlignHCenter
 
-            Milou.ResultsListView {
+            Milou.ResultsView {
                 id: listView
                 queryString: queryField.text
                 clip: true
@@ -158,7 +158,7 @@ MouseArea {
                         }
                     }
 
-                    // Used by ResultsListView to determine next tab action
+                    // Used by ResultsView to determine next tab action
                     function activateNextAction() {
                         queryField.forceActiveFocus();
                         queryField.selectAll();
-- 
2.47.3

